plugin/pom.xml

```diff
@@ -24,13 +24,14 @@ THE SOFTWARE.
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>io.jenkins.plugins</groupId>
         <artifactId>redis-fingerprint-storage-parent</artifactId>
         <version>1.0-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
```

Comment:
I do not see any reason why sources are in a `plugin/` submodule. 

```xml
    <modules>
        <module>plugin</module>
    </modules>
```

indicates that you could just move everything to the root.

Reply:
For future reference: https://github.com/jenkinsci/archetypes#usage

Reply:
@jglick Thanks a lot for reviewing! We chose to start with a multimodule repository because we decided that it might help us in the future.
CC @oleg-nenashev

Reply:
YAGNI. If and when there is a need for another submodule (for example because you need to shade some library dependency), you can move existing sources into a submodule in the same PR which introduces the new submodule and an aggregator POM.

Reply:
@jglick we discussed it at the team meeting, and we agreed that it is better to have a submodule for now. Yes, just in case we need that (Irespectfully disagree with YAGNI). It does not really matter for alpha releases, and we can decide on the refactoring when we get closer to GA.

Reply:
Whatever you prefer, just a recommendation.